guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc

```diff
@@ -21,25 +26,28 @@ endif::[]
 
 Infrastructure and host management services:
 
-* *DHCP* – {SmartProxy} can act as a DHCP server or it can integrate with an existing solution, including ISC DHCP servers, Active Directory, and Libvirt instances.
+* *DHCP* – {SmartProxy} can manage a DHCP server, including integration with an existing solution such as ISC DHCP servers, Active Directory, and Libvirt instances.
 
-* *DNS* – {SmartProxy} can act as a DNS server or it can integrate with an existing solution, including ISC BIND and Active Directory.
+* *DNS* – {SmartProxy} can manage a DNS server, including integration with an existing solution such as ISC BIND and Active Directory.
 
-* *TFTP* – {SmartProxy} can act as a TFTP server or integrate with any UNIX-based TFTP server.
+* *TFTP* – {SmartProxy} can integrate with any UNIX-based TFTP server.
 
 * *Realm* – {SmartProxy} can manage Kerberos realms or domains so that hosts can join them automatically during provisioning.
 {SmartProxy} can integrate with an existing infrastructure, including {FreeIPA} and Active Directory.
 
 * *Puppet server* – {SmartProxy} can act as a configuration management server by running Puppet server.
 
-* *Puppet Certificate Authority* – {SmartProxy} can act as a Puppet CA to provide certificates to hosts.
+* *Puppet Certificate Authority* – {SmartProxy} can integrate with Puppet's CA to provide certificates to hosts.
 
-* *Baseboard Management Controller (BMC)* – {SmartProxy} can provide power management for hosts.
+* *Baseboard Management Controller (BMC)* – {SmartProxy} can provide power management for hosts using IPMI or Redfish.
 
 * *Provisioning template proxy* – {SmartProxy} can serve provisioning templates to hosts.
 
+ifndef::foreman-deb::[]
 * *OpenSCAP* – {SmartProxy} can perform security compliance scans on hosts.
+endif::[]
```

Comment:
You can still install from rubygems. No need to hide this I think.

Reply:
I'd prefer to not support that. I really don't want to support gem installations of Smart Proxy plugins. Either we package & support it, or we don't.